README.md

```diff
@@ -126,3 +126,15 @@ Django Settings
 AssertRegex
 * From [StackOverflow](https://stackoverflow.com/a/64192098)
 * Accessed Feb 14 2021
+
+Django Restframework
+* From [Bennett Garner](https://medium.com/swlh/build-your-first-rest-api-with-django-rest-framework-e394e39a482c)
```

Comment:
Can we seperate this to 
* From [Bennett Garner](their medium profile page)
* From [Medium](link to where you found it)